src/local-development/helpers/fs.ts

```diff
@@ -0,0 +1,19 @@
+import * as vscode from 'vscode';
+import { Uri } from 'vscode';
+
+export async function exists(uri: vscode.Uri): Promise<boolean> {
+	try {
+		await vscode.workspace.fs.stat(uri);
+		return true; // File exists
+	} catch (error) {
+		if (error instanceof vscode.FileSystemError && error.code === 'FileNotFound') {
+			return false; // File does not exist
+		}
+		// Handle other errors if necessary
+		throw error;
+	}
+}
+
+export async function removeRecursively(uri: Uri) {
+	await vscode.workspace.fs.delete(uri, { recursive: true });
+}
```

Comment:
Consider if it is useful to wrap one-line function call by another function here.

Reply:
renamed to delete with options, wrapped due to feature gh actions